b2sdk/bucket.py

```diff
@@ -369,6 +371,7 @@ def ls(
         recursive: bool = False,
         fetch_count: int | None = LIST_FILE_NAMES_MAX_LIMIT,
         with_wildcard: bool = False,
+        filters: Sequence[Filter] | None = None,
```

Comment:
This reminds me of recent axel's blog post where he suggests using `filters: Sequence[Filter] = ()` in such cases. Since tuples are immutable, works out okay.

(Not a request for change, just something I found curious comment.)

Reply:
It never crossed my mind! That looks way better!